pkg/metaserver/application_test.go

```diff
@@ -0,0 +1,409 @@
+package metaserver
+
+import (
+	"context"
+	"crypto/sha256"
+	"fmt"
+	"testing"
+
+	"github.com/kubeedge/beehive/pkg/core/model"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/runtime/schema"
```

Comment:
The import should be sorted by format: Standard lib; third-party lib; local package